app/assets/stylesheets/top-bar.scss

```diff
@@ -67,6 +79,12 @@
   @media (min-width: $breakpoint-s) {
     margin: 0 var(--su-4);
   }
+
+  &--logged-out {
+    @media (min-width: $breakpoint-s) {
+      max-width: $breakpoint-s;
+    }
+  }
```

Comment:
Don't you think it looks a bit off when search is different when logged in vs logged out? Of course its just my personal opinion but why do we do that in the first place?

Reply:
I think it looks nicer if there's space in the center to have the space bar align with then main feed column — less jumpy eye movement :)